### PR TITLE
mount a login page for profile creation

### PR DESCRIPTION
**Pull request by pohl**
- [x] create a page at `/login` to hold the form for login/user-creation
- [ ] style the form correctly
- [ ] make the component post 
- [ ] if the user already existed, show `/profile/:id`
- [ ] all the things

### CODE REVIEW

File: src/app/login.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Layout from './layout';
+
+var UserLogin = React.createClass({
```
Comment (JDegner0129) [inline src/app/login.js]: You should use ES6-style classes here. See [this doc](https://facebook.github.io/react/docs/reusable-components.html#es6-classes)

File: src/app/login.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  getInitialState: function() {
```
Comment (JDegner0129) [inline src/app/login.js]: In ES6, you can do `getInitialState() { ... }` instead of explicitly declaring a function

Comment (pohl) [inline src/app/login.js]: Does that apply to all of the functions within a class (`render`, etc.)?

File: src/app/login.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Layout from './layout';
+
+class UserLogin extends React.Component {
+  getInitialState() {
```
Comment (JDegner0129) [inline src/app/login.js]: FWIW, `getInitialState` won't work with ES6 React components. You'll want to do something like:

``` javascript
constructor() {
  this.state = { ... };
}
```

**Pull request closed without merging**